handlers/create.go

```diff
@@ -79,23 +83,42 @@ func CreateMachine(event *events.Event, apiClient *client.RancherClient) error {
 	return publishReply(reply, apiClient)
 }
 
-func logProgress(resourceId string, readerStdout io.Reader, readerStderr io.Reader) {
+func logProgress(readerStdout io.Reader, readerStderr io.Reader, uuid string, event *events.Event, apiClient *client.RancherClient) {
 	// We will just log stdout first, then stderr, ignoring all errors.
 	scanner := bufio.NewScanner(readerStdout)
 	for scanner.Scan() {
+		msg := scanner.Text()
 		log.WithFields(log.Fields{
-			"resourceId: ": resourceId,
-		}).Infof("stdout: %s", scanner.Text())
+			"resourceId: ": event.ResourceId,
+		}).Infof("stdout: %s", msg)
+		msg = filterDockerMessage(msg, uuid)
+		if msg != "" {
+			publishTransitioningReply(msg, event, apiClient)
+		}
 	}
 
 	scanner = bufio.NewScanner(readerStderr)
 	for scanner.Scan() {
 		log.WithFields(log.Fields{
-			"resourceId": resourceId,
+			"resourceId": event.ResourceId,
 		}).Infof("stderr: %s", scanner.Text())
 	}
 }
 
+func filterDockerMessage(msg string, uuid string) string {
+	// Docker log messages come in the format: time=<t> level=<log-level> msg=<message>
+	// We just want to return <message> to cattle and only messages that do not contain the machine uuid
+	re := regexp.MustCompile("msg=.*")
```

Comment:
Compile this regex at the package level so it only needs done once, like this:
https://github.com/rancherio/go-machine-service/blob/master/handlers/common.go#L18